webapp/WEB-INF/conf/plugins/workflow-actionsbatch.properties

```diff
@@ -4,3 +4,5 @@
 # Default Labels for XPage
 workflow-actionsbatch.pageTitle=workflow-actionsbatch
 workflow-actionsbatch.pagePathLabel=workflow-actionsbatch
+
+workflow-actionsbatch.wait.time=1000
```

Comment:
Name this property : workflow-actionsbatch.pause.ms
put (-1) as default value
and add a comment like "Pause time between each action in MS"

Reply:
could you please add a comment and the default value ?